python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -393,68 +345,50 @@ def checkShouldFlush(self):
             self._flushLastTime = timeNow
             self.dirty_hits = 0
 
-        self._lock.release()
-
         return decision
 
     def _getObjects(self):
         logger.debug("_getObjects")
-        #self._lock.acquire()
         returnable = self._objects
-        #self._lock.release()
         return returnable
 
     def ids(self):
```

Comment:
OK, I guess we can wait for 6.1.17 for a review of what should or shouldn't be locked...